File: c/questao8.c
#include <stdio.h>

int main() {
    char opcao;
    float num1, num2, resultado;

    printf("Digite os dois números separados por espaço: ");
    scanf("%f %f", &num1, &num2);

    printf("Digite a operação desejada (+, -, *, /): ");
    scanf("%s", &opcao);

    switch (opcao)
    {
    case '+':
        resultado = num1 + num2;
        printf("O resultado é: %.1f\n", resultado);
        break;
    case '-':
        resultado = num1 - num2;
        printf("O resultado é: %.1f\n", resultado);
        break;
    case '*':
        resultado = num1 * num2;
        printf("O resultado é: %.1f\n", resultado);
        break;
    case '/':
        resultado = num1 / num2;
        printf("O resultado é: %.1f\n", resultado);
        break;
    default:
        printf("Opção inválida!");
        break;
    }

    return 0;
}

File: python/questao8.py
def main():
    opcao = input("Digite a operação desejada (+, -, *, /): ")
    num1, num2 = map(float, input("Digite os dois números separados por espaço: ").split())

    if opcao == '+':
        resultado = num1 + num2
        print(f"O resultado é: {resultado:.1f}")
    elif opcao == '-':
        resultado = num1 - num2
        print(f"O resultado é: {resultado:.1f}")
    elif opcao == '*':
        resultado = num1 * num2
        print(f"O resultado é: {resultado:.1f}")
    elif opcao == '/':
        if num2 != 0:
            resultado = num1 / num2
            print(f"O resultado é: {resultado:.1f}")
        else:
            print("Não é possível dividir por zero.")
    else:
        print("Opção inválida!")

if __name__ == "__main__":
    main()
